xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/importer/import.css

```diff
@@ -71,9 +66,7 @@ div.loading {
 
 /* Package Content styles */
 #packagecontainer {
-  float: left;
   margin-left: 2%;
```

Comment:
if we rely on the BS grid, than we should remove all the properties that are changing margins, width, etc.